### PR TITLE
perf(biblio-ref): add premium crossref token

### PR DESCRIPTION
**Pull request by leogail**

### CODE REVIEW

**Comment by leogail** [comment]
Les tests ne passent pas : il faut ajouter la variable d'environnement CROSSREF_API_KEY dans les secrets github

File: services/biblio-ref/README.md [hunk 1]
```diff
@@ -6,21 +6,6 @@ Si un DOI est trouvé dans la référence bibliographique, valide la référence
 
 ## Construction de l'image docker
 
-Nécessite les variables d'environnement:
+Nécessite une variable d'environnement `API_TOKEN`, un token pour utiliser l'API Crossref. Elle doit se trouver dans un un fichier `.env` à la racine.
```
Comment (parmentf) [inline services/biblio-ref/README.md:9]: **issue:** I thought the env var was `CROSSREF_API_KEY`, not `API_TOKEN`...‏

**Review by parmentf — CHANGES_REQUESTED** [review]
Improve your README.md.

We still have to add `CROSSREF_API_KEY` as a secret variable, for the tests to pass.

File: services/biblio-ref/README.md (hunk 1 above)
Comment (leogail) [reply services/biblio-ref/README.md:9]: I changed it after seeing how you names openalex env var but forgot it in readme. Done.

Comment (leogail) [inline services/biblio-ref/README.md:9]: **praise:** 
```suggestion
Nécessite une variable d'environnement `CROSSREF_API_KEY`, un token pour utiliser l'API Crossref. Elle doit se trouver dans un un fichier `.env` à la racine (voir le fichier .env.example associé)
```

**Review by parmentf — APPROVED** [review]

File: services/biblio-ref/config.json [hunk 2]
```diff
@@ -3,13 +3,14 @@
         "EZS_TITLE": "Valide une référence bibliographique",
         "EZS_DESCRIPTION": "Trouve les références bibliographiques d'un PDF puis utilise crossref pour: - valider les références bibliographiques - donner les DOIs s'ils existent - indiquer les éventuelles références rétractées.",
         "EZS_METRICS": true,
-        "EZS_CONCURRENCY": 1,
+        "EZS_CONCURRENCY": 3,
```
Comment (parmentf) [inline services/biblio-ref/config.json:6]: **note (non-blocking):** 3 is still very large for `EZS_CONCURRENCY`

This variable is used to prepare enough process to answer to many requests, not to execute many processes for one request. I think you misunderstood something.‏

**Pull request merged**